.azure-pipelines/libffi-build.yml

```
@@ -13,7 +13,7 @@ jobs:
 - job: Build_LibFFI
   displayName: LibFFI
   pool:
-    vmImage: windows-latest
+    vmImage: windows-2022
```

I'm not sure about this one.  This particular job should never actually be run again, though, so I'm not sure how much it matters either way.

---

Yeah, I meant to leave out the AP ones. Let's revert:
```suggestion
    vmImage: windows-latest
```